debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -575,33 +575,33 @@ private String replaceDatabaseNamePlaceholder(String sql, String databaseName) {
     public void callbackOnReadingNewChangeTable(SqlServerPartition partition, ChangeTable table) throws SQLException {
         final String query = replaceDatabaseNamePlaceholder(START_READING_CHANGE_TABLE, partition.getDatabaseName());
         prepareUpdate(query, ps -> {
-            LOGGER.trace("Calling the StartReadingFromCaptureInstance stored procedure with change table: {}", table);
+            LOGGER.info("Calling the StartReadingFromCaptureInstance stored procedure with change table: {}", table);
```

Comment:
I'd say it's more of a debug than info. Info is something relevant from the business logic perspective while debug is an implementation detail.